.github/workflows/linux.yml

```diff
@@ -91,8 +91,18 @@ jobs:
           set -x
           git fetch --prune --unshallow
           export VERSION=$(git describe --abbrev=4)
+          git fetch --depth 1 # drop history
```

Comment:
I believe we don't need to drop history any more?

Reply:
Thanks for catching that! All cleaned up now (dropped the sample virus commit).